docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/how-to-do-the-transition.md

```diff
@@ -0,0 +1,46 @@
+---
+title: How to do the transition?
+description: This document describes how to do the transition.
+template: howto-guide-template
+---
+
+# How to do the transition?
+
+{% info_block infoBox %}
+
+Resources: Backend, DevOps
+
+{% endinfo_block %}
+
+## Description
+
+In this step, we need to assess efforts related to the exact moment of switching the shops from on-prem to Spryker Cloud.
+
+Before you start the transition please make sure that:
+
+1. Depending on a environment type a correspondent template was applied in your cloud service provider (currently, is AWS).
+    All stage similar ones (dev/uat/test/qa etc) should have only one run task per basic service (Yves/Zed/Glue...).
```

Comment:
This looks incorrect. Can you elaborate on the meaning?

Reply:
This is correct statement. Only prod infra template uses 2 task per service.

Reply:
Task is a docker container in terms of ECS definition.